packages/components/src/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Children } from 'react';
+import { ComponentProps } from '@marigold/types';
+import flattenChildren from 'react-keyed-flatten-children';
+
+import { Close } from '@marigold/icons';
+import { Button } from '../Button';
+import { Box } from '../Box';
+
+export type DialogProps = {
+  variant?: string;
+  onClose?: ComponentProps<typeof Button>['onClick'];
+} & ComponentProps<'div'>;
+
+export const Dialog: React.FC<DialogProps> = ({
+  variant = 'basic',
+  onClose,
+  children,
+  className,
+  ...props
+}) => {
+  let dialogItems = flattenChildren(children);
+
+  return (
+    <Box display="flex" width="100%">
+      <Box
+        {...props}
+        display="block"
+        borderRadius="2px"
```

Comment:
Das sollte von der `theme` kommen. So kann man es nicht ändern.

Reply:
Bze wenn eine Theme ein border-radius will, dann kommt der doch über die Variant! :)

Reply:
Dasselbe gilt für die Paddings.

Reply:
Wie mache ich das? Ich habe ja mehrere Boxes aber kann nur bei einer die variant hinterlegen

Reply:
Kommt drauf an. Machen mehrere `variant` bei einem Dialog überhaupt Sinn? Es würde ja auch eine reichen, oder? Dann könnte man das Lösen über `dialog.wrapper`, `dialog.body` etc.

Reply:
Ja das könnte man auf jeden Fall, hab daran jetzt nicht gedacht weil wir das noch bei keiner Component so gemacht haben.

Reply:
Ja. Wahrscheinlich sollten das ein paar mehr haben? Wir haben die Variant ja bewusst (auch durch das Entfernen des System helpers) optional gemacht damit etwas freier sind was die APIs angeht :)

Reply:
Aber BTW. Ich glaube es ist einfacher nur den Dialog zu machen. Nicht schon mit dem Titel usw. Das kann man auch in Userland machen. Es geht ja vorrangig um die Funktionalität eines Dialogs!